src/main/java/com/github/fishio/MenuField.java

```diff
@@ -0,0 +1,41 @@
+package com.github.fishio;
+
+import java.util.ArrayList;
+
+import javafx.scene.canvas.Canvas;
+
+/**
+ * Extended class of playingfield that renders fish on the background of the main manu.
+ */
+public class MenuField extends PlayingField {
+
+	/**
+	 * Constructor for the menuField.
+	 * @param fps
+	 * 		the amount of frames per second of the render thread.
+	 * @param canvas
+	 * 		the canvas to render on.
+	 */
+	public MenuField(int fps, Canvas canvas) {
+		super(fps, canvas, 0);
+		}
+
+	@Override
+	public ArrayList<PlayerFish> getPlayers() {
+		return null;
```

Comment:
Why return null? Just return an empty arrayList, that would make more sense.

<logic